kpet/targeted.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2018 Red Hat, Inc. All rights reserved. This copyrighted
+# material is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General Public
+# License v.2 or later.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Module where targeted testing is implemented"""
```

Comment:
This is quite redundant. It's already a module, and something is obviously implemented here. So just """Targeted testing""" would be fine, and then adding some explanation about what it actually is would be even better. Something like: """Targeted testing - matching patches to applicable test cases""", or such.

Not necessary for my ACK.

Reply:
Done